stories/TextArea.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import { boolean, select, text, withKnobs } from '@storybook/addon-knobs'; // eslint-disable-line import/no-extraneous-dependencies
+import TextArea from '../src/components/TextArea';
+import { CONTEXTS, SIZES } from '../src/constants';
+
+storiesOf('TextArea', module)
+    .addDecorator(withKnobs)
+    .add('Default behavior', () => (
+        <TextArea
+            context={select('Context', CONTEXTS, CONTEXTS[1])}
+            defaultValue="This is a default value"
+            disabled={boolean('Disabled', false)}
+            isBlock={boolean('isBlock', false)}
+            placeholder={text('Placeholder', 'Some text goes here...')}
+            onChange={e => {
+                // eslint-disable-next-line no-console
+                console.log(e.target.value);
+            }}
+            size={select('Size', SIZES, SIZES[1])}
+        />
+    ))
+    .add('Controlled component', () => (
+        <TextArea
+            context={select('Context', CONTEXTS, CONTEXTS[1])}
+            disabled={boolean('Disabled', false)}
+            isBlock={boolean('isBlock', false)}
+            onChange={e => {
```

Comment:
The function here and in the other story are the same, but written differently. Consider writing them the same, so that it is clear that this is not the main point. Also might be good to list the props in the same order in both cases.